### PR TITLE
feat: Add guide for Mint NFT page

### LINKED ISSUE #314: Feature Request: Guide for Mint NFT Page
### Describe the solution you'd like

We need to develop the Guide section for the Mint NFT page. Here are some requirements:

- Use the [Figma](https://www.figma.com/file/faBZpJPdUwOjcP9l0usY3a/Build-Onchain-Apps?type=design&node-id=175-20778&mode=design&t=6AHHLpqDZSlQotYv-4) as a guide for the frontend UI
- Use the CodeBlock component to show code snippets
- Explain the goal of the experience
- Explain how the contract works and the main operations
- Explain how we interact with the contract on the frontend

### Describe alternatives you've considered.

_No response_

### PR DESCRIPTION
**Pull request by ppiyush13**
**What changed? Why?**
- Add guide for Mint NFT page
- Fixes #314
- [Figma link](https://www.figma.com/file/faBZpJPdUwOjcP9l0usY3a/Build-Onchain-Apps?type=design&node-id=175-20778&mode=design&t=6AHHLpqDZSlQotYv-4)

**How has it been tested?**
- Manually on local

**Notes to reviewers**

## Desktop design
Screenshot 2024-02-08 at 12 51 10 AM

## Mobile desgin
Screenshot 2024-02-08 at 12 52 44 AM

### CODE REVIEW

File: template/app/mint/index.tsx [hunk 1]
```diff
@@ -25,7 +26,9 @@ export default function MintPage() {
       <Header />
       <main className="container mx-auto flex flex-col px-8 py-6">
         <Banner pageName="Mint NFT" pageUrl="mint" />
+        <HorizontalLine />
         <MintContractDemo />
+        <HorizontalLine />
```
Comment (ppiyush13) [inline template/app/mint/index.tsx:31]: Missing as per [Figma](https://www.figma.com/file/faBZpJPdUwOjcP9l0usY3a/Build-Onchain-Apps?node-id=402%3A5077&mode=dev) designs. Added

**Comment by arsood** [comment]
Correct me if I'm wrong, but wasn't this guide supposed to be specific to the Mint NFT flow? I think the Figma is simply a design guide and not the actual content we want here. @Zizzamia can you confirm?

**Comment by Zizzamia** [comment]
@arsood is right!

**Comment by arsood** [comment]
Closing for now. Happy to review again if/when content is added.

**Pull request closed without merging**